PRESIGNED-PAUSE.md

```diff
@@ -40,6 +42,19 @@ doesn’t even need to be funded. These are offchain signatures.
 
 A Facilitator will collect the signatures and execute the contract.
 
+### Ensure no gaps with PSPs coverage
+
+We need to ensure there is no gaps in the PSPs coverage during upgrade.
+This means that, if we are upgrading core components of the system. It can break the PSPs coverage, thus we need to ensure before the upgrade that the PSPs coverage will not be broken.
```

Comment:
```suggestion
This means that, if we are upgrading core components of the system, it can break the PSPs coverage, thus we need to ensure before the upgrade that the PSPs coverage will not be broken.
```